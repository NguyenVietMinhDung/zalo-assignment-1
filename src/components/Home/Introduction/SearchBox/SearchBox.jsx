import React from 'react';
import './SearchBox.scss';

const SearchBox = () => (
  <div className="search-box">
    <div className="search-box__search-icon">
      <i className="lni lni-16 lni-search-alt" />
    </div>
    <input type="text" placeholder="Tìm kiếm tên sản phẩm" />
  </div>
);

export default SearchBox;
