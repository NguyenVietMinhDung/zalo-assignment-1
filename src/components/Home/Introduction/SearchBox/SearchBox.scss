@import '../../../../styles/variables';

.search-box {
	display: flex;

	&__search-icon {
		position: absolute;

		display: flex;

		width: 50px;
		height: 100%;

		cursor: pointer;
		align-items: center;
		justify-content: center;

		i {
			color: lightgray;

			font-weight: bold;
		}
	}

	input {
		width: 100%;
		padding: 10px 10px 10px 50px;

		border: 1px solid $dark-white-smoke;
		border-radius: 50px;

		background-color: $white-smoke;

		&::placeholder {
			color: lightgray;

			font-weight: bold;
		}

		&:focus {
			outline: none;
		}
	}
}
