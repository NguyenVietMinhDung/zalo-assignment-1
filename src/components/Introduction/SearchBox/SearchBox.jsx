import React from 'react';
import { Container, Row, Col } from 'react-grid-system';
import './Introduction.scss';
import banner from '../../assets/images/BANNER.png';

const SearchBox = () => (
  <Container>
    <Row className="introduction">
      <Col className="introduction__banner" xs={12} style={{ padding: 0 }}>
        <img src={banner} alt="" />
      </Col>
    </Row>
  </Container>
);

export default SearchBox;
