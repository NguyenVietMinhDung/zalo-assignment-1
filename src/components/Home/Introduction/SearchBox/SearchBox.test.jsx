import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './SearchBox';

describe('<SearchBox />', () => {
  test('renders correctly', () => {
    const component = shallow(<SearchBox />);
    expect(component).toMatchSnapshot();
  });
});
