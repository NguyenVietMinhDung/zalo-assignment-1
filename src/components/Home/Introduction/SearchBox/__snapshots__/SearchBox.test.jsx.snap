// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> renders correctly 1`] = `
<div
  className="search-box"
>
  <div
    className="search-box__search-icon"
  >
    <i
      className="lni lni-16 lni-search-alt"
    />
  </div>
  <input
    placeholder="Tìm kiếm tên sản phẩm"
    type="text"
  />
</div>
`;
